docs/guides/kubernetes/getting-started-with-load-balancing-on-a-lke-cluster/index.md

```diff
@@ -201,6 +201,10 @@ tls.key:  1704 bytes
 
 #### Configuring TLS within a Service
 
+{{< note >}}
```

Comment:
@Rajakavitha1 I think this note is a little bit unclear. When you say "exposes the endpoints" it's not clear what you're referring to. Similarly, when you say "when you access it from inside the cluster". 

One suggestion: "By default, Kubernetes does not expose Services with TLS termination over HTTPS." I might remove the note and just add the suggested line to the beginning of the intro paragraph of this section.

Do you mean communication between nodes and the control plane? If yes, here is some documentation with information: https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/ <--

Reply:
Fixed